test/turbopack-dev-tests-manifest.json

```diff
@@ -8708,53 +8807,49 @@
     "runtimeError": false
   },
   "test/integration/create-next-app/package-manager/bun.test.ts": {
-    "passed": [
-      "should use bun for --use-bun flag with example",
-      "should use bun when user-agent is bun",
-      "should use bun when user-agent is bun with example"
-    ],
+    "passed": [],
```

Comment:
`describe` above them was skipped in #65845

https://github.com/vercel/next.js/blame/af49a5b54c06fc2b5c505fc4bce2dd64657b8551/test/integration/create-next-app/package-manager/bun.test.ts#L29

Reply:
Those are `create-next-app` tests, and at #65926, I moved some `it` tests that were outside the `describe` function without intended (I know it because [I'm the one that made a mistake](https://github.com/vercel/next.js/pull/63921) 😅).

As mentioned above, the whole tests were `describe.skip`, and should've moved to `pending` before!